dot/state/epoch_test.go

```diff
@@ -591,12 +597,24 @@ func TestStoreAndFinalizeBabeNextConfigData(t *testing.T) {
 			err := epochState.blockState.db.Put(headerKey(tt.finalizeHash), []byte{})
 			require.NoError(t, err)
 
+			// before finalize next config data this epoch should not be defined
```

Comment:
nit maybe a word missing I think?

```suggestion
			// before finalize next config data for this epoch should not be defined
```